app/views/users/edit.html.haml

```diff
@@ -0,0 +1,48 @@
+%h1 Edit Profile
+= form_for @user do |f|
+  .row
+    .input-field.col.s12
+      = f.label :email, :class => "active"
+      = f.email_field :email, disabled: true, autocomplete: "off"
+  .row
+    .input-field.col.s12
+      = f.label :name
+      = f.text_field :name, autofocus: true, autocomplete: "off"
+  .row
+    .file-field.input-field.col.s12
+      .btn
+        %span Image
+        = f.file_field :photo, autocomplete: "off"
+      .file-path-wrapper
+        %input.file-path.validate{ :type => "text" }
+  .row
+    .input-field.col.s6
+      = f.label :password
+      = f.password_field :password, autocomplete: "off"
+    .input-field.col.s6
+      = f.label :password_confirmation
+      = f.password_field :password_confirmation, autocomplete: "off"
+  .row
+    .input-field.col.s12
+      = f.label :current_password
+      = f.password_field :current_password, autocomplete: "off"
+
+  %br
+  %br
+  .row
+    .col.s2
+      -# = f.submit :class => 'btn waves-effect waves-light'
+      = button_tag( :class => "btn waves-effect waves-light" ) do
+        Update
+        %i.material-icons.right send
+
+- if !@user.has_role?(:admin)
+  %br
+  %br
+  %br
+  %h2 Cancel Account User
+  = form_tag({:controller => "users", :action => "destroy", :id => @user}, method: "delete") do
```

Comment:
After ruby `1.9` this hash rocket/fat arrow syntax can be updated as `something: ` instead of `:something =>` and update all remaining data